integrated_channels/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Integrated channel package for Enterprises.
```

Comment:
Can you provide a little more context here around what "integrated channels" are?  Also I'd say "[...] for the Open edX Enterprise Service."